tsfc/coffee.py

```diff
@@ -296,7 +299,7 @@ def _expression_scalar(expr, parameters):
     if isnan(expr.value):
         return coffee.Symbol("NAN")
     else:
-        return coffee.Symbol(("%%.%dg" % (PRECISION - 1)) % expr.value)
+        return coffee.Symbol(("%%.%dg" % (parameters.precision - 1)) % expr.value)
```

Comment:
I don't quite remember why this minus one here. It seems to mimic `epsilon`, but `epsilon` is used as an absolute tolerance for rounding, while with `%g` this means a relative tolerance. Maybe I wasn't aware of this distinction?